docs/en/install-upgrade/release-notes/release-notes-logstash-rc1.asciidoc

```diff
@@ -1,3 +1,19 @@
 = {ls} version 9.0.0-rc1
 
-coming::[9.0.0-rc1]
\ No newline at end of file
+coming::[9.0.0-rc1]
+
+[[logstash-900-rc1-features-enhancements]]
+== Features and enhancements 
+
+* Use UBI9 as base image https://github.com/elastic/logstash/pull/17174[#17174]
+* Improve plugins remove command to support multiple plugins https://github.com/elastic/logstash/pull/17030[#17030]
+* Allow concurrent Batch deserialization https://github.com/elastic/logstash/pull/17050[#17050]
+* Upgrade filter-xml and its nokogori dependency https://github.com/elastic/logstash/pull/17130[#17130]
+* Update rack dependency to 3.1.11 https://github.com/elastic/logstash/pull/17230[#17230]
+* Upgrade elasticsearch-ruby client to v8 https://github.com/elastic/logstash/pull/17161[#17161]
```

Comment:
@mashhurs:  It looks like the elasticsearch-ruby client is stack-versioned, with [current](https://www.elastic.co/guide/en/elasticsearch/client/ruby-api/current/index.html) being 8.17. Should we be more precise than v8?